### ISSUE TITLE
Saving and loading for all model variations

### ISSUE THREAD

**Issue by brian6091** [brian6091/Dreambooth#26]
https://discuss.pytorch.org/t/how-to-save-the-requires-grad-state-of-the-weights/52906

https://pytorch.org/tutorials/recipes/recipes/warmstarting_model_using_parameters_from_a_different_model.html

**Comment by brian6091** [comment]
https://androidkt.com/how-to-copy-pytorch-tensor-using-clone-detach-and-deepcopy/
https://stackoverflow.com/questions/62437509/what-is-the-difference-between-detach-clone-and-deepcopy-in-pytorch-tensors-in
https://androidkt.com/copy-pytorch-model-using-deepcopy-and-state_dict/
https://ofstack.com/python/40554/solve-the-state_dict-of-copy-problem-of-pytorch.html

**Comment by brian6091** [comment]
https://ofstack.com/python/39062/detailed-explanation-of-copy-and-local-operation-in-pytorch.html
